File: Angular-clientSize/tripAngular/src/app/classes/Trip.ts
export class Trip{
    constructor(
         public  idTrip?:number,
         public destinationTrip?:string,
         public idType?:number,
         public dateTrip?:Date,
         public leavingTime?:number,
         public durationTrip?:number,
         public placesAvailable?:number,
         public price?:number,
         public image?:string,
         public typeName?:string,
         ) {
    }
}

File: C#-serverSize/DTO/classes/Trip.cs
using AutoMapper;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace DTO.classes
{
    public partial class Trip
    {
        public short IdTrip { get; set; }

        public string? DestinationTrip { get; set; }

        public short? IdType { get; set; }

        public DateTime? DateTrip { get; set; }

        public short? LeavingTime { get; set; }

        public short? DurationTrip { get; set; }

        public short? PlacesAvailable { get; set; }

        public short? Price { get; set; }

        public string? Image { get; set; }
        public string? typeName { get; set; }

        

    }

}